client/state/plugins/installed/selectors-ts.ts

```diff
@@ -446,3 +446,25 @@ export const getPluginStatusesByType = createSelector(
 	},
 	( state: AppState ) => state.plugins.installed.status
 );
+
+/**
+ * Returns a Boolean indicating if a site has a particular plugin that
+ * is active. This is useful for jetpack connected sites.
+ *
+ * @param {Object} state - Global state tree
+ * @param {number} siteId - Site ID
+ * @param {string} pluginSlug - Plugin slug
+ * @returns {boolean} - truthiness of a site having an active plugin
```

Comment:
Minor suggestion for improving clarity

```suggestion
 * @returns {boolean} - True if that plugin is active for that site, false otherwise.
```